go.mod

```diff
@@ -1,126 +1,84 @@
+replace github.com/go-ldap/ldap/v3 => github.com/go-ldap/ldap/v3 v3.4.3
 
-replace github.com/dgrijalva/jwt-go v3.2.0+incompatible => github.com/golang-jwt/jwt v3.2.2+incompatible
-
-replace (
-	github.com/containerd/containerd => github.com/containerd/containerd v1.6.6
-	github.com/go-ldap/ldap/v3 => github.com/go-ldap/ldap/v3 v3.4.3
-	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
-	github.com/hashicorp/consul => github.com/hashicorp/consul v1.12.2
-	github.com/hashicorp/consul/api => github.com/hashicorp/consul/api v1.13.0
-	github.com/hashicorp/consul/sdk => github.com/hashicorp/consul/sdk v0.9.0
-	github.com/hashicorp/vault/api => github.com/hashicorp/vault/api v1.7.2
-	github.com/spf13/viper => github.com/spf13/viper v1.8.1
-	github.com/ulikunitz/xz => github.com/ulikunitz/xz v0.5.10
-	golang/github.com/opencontainers/runc => golang/github.com/opencontainers/runc v1.1.3
-	golang/github.com/pkg/sftp => golang/github.com/pkg/sftp v1.13.5
-	golang/github.com/valyala/fasthttp => golang/github.com/valyala/fasthttp v1.37.0
-// k8s.io/api => k8s.io/api v0.0.0-20190325185214-7544f9db76f6
-// k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.22.1
-// k8s.io/apimachinery => k8s.io/apimachinery v0.22.1
-// k8s.io/apiserver => k8s.io/apiserver v0.22.1
-// k8s.io/cli-runtime => k8s.io/cli-runtime v0.22.1
-// k8s.io/client-go => k8s.io/client-go v1.5.2
-// k8s.io/cloud-provider => k8s.io/cloud-provider v0.22.1
-// k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.22.1
-// k8s.io/code-generator => k8s.io/code-generator v0.22.1
-// k8s.io/component-base => k8s.io/component-base v0.22.1
-// k8s.io/component-helpers => k8s.io/component-helpers v0.22.1
-// k8s.io/controller-manager => k8s.io/controller-manager v0.22.1
-// k8s.io/cri-api => k8s.io/cri-api v0.22.1
-// k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.22.1
-// k8s.io/gengo => k8s.io/gengo v0.0.0-20210813121822-485abfe95c7c
-// k8s.io/klog/v2 => k8s.io/klog/v2 v2.20.0
-// k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.22.1
-// k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.22.1
-// k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20210817084001-7fbd8d59e5b8
-// k8s.io/kube-proxy => k8s.io/kube-proxy v0.22.1
-// k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.22.1
-// k8s.io/kubectl => k8s.io/kubectl v0.22.1
-// k8s.io/kubelet => k8s.io/kubelet v0.22.1
-// k8s.io/kubernetes => k8s.io/kubernetes v1.22.6
-// k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.22.1
-// k8s.io/metrics => k8s.io/metrics v0.22.1
-// k8s.io/mount-utils => k8s.io/mount-utils v0.22.1
-// k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.22.1
-// k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.22.1
-// k8s.io/system-validators => k8s.io/system-validators v1.5.0
-// k8s.io/utils => k8s.io/utils v0.0.0-20210820185131-d34e5cb4466e
-)
+replace github.com/hashicorp/consul => github.com/hashicorp/consul v1.13.3
```

Comment:
We should not be using `replace` for dependency upgrading. Use `go get -u github.com/hashicorp/consul` or `go get github.com/hashicorp/consul@v1.13.3` instead. `replace` is intended for replacing module dependencies for local development.

Reply:
NB. I appreciate this incorrect precedent has already been set by others in this module, but we need to undo it as this is not the correct way to upgrade transient dependencies. One of the worst side effects of using `replace` in this way is that module authentication (go.sum) doesn't take place which might present a security risk. 
See the note at the end of this section… https://go.dev/doc/modules/managing-dependencies#local_directory
> Note: When you use the replace directive, Go tools don’t authenticate external modules as described in [Adding a dependency](https://go.dev/doc/modules/managing-dependencies#adding_dependency).